Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Core/Classes/UAC/UACUtilityMethods.cs

```diff
@@ -1,29 +1,21 @@
-﻿#region BSD License
-/*
- * Use of this source code is governed by a BSD-style
- * license or other governing licenses that can be found in the LICENSE.md file or at
- * https://raw.githubusercontent.com/Krypton-Suite/Extended-Toolkit/master/LICENSE
- */
-#endregion
-
-namespace Krypton.Toolkit.Suite.Extended.Buttons
+﻿namespace Krypton.Toolkit.Suite.Extended.Core
 {
-    /// <summary>Provides useful methods for specified tasks.</summary>
-    public class UtilityMethods
+    /// <summary>Utility methods for the user account control.</summary>
+    public class UACUtilityMethods
```

Comment:
Make this an internal, so that Name changes to it do not break Exposed public callers

Reply:
Done